auto/modules/nodejs

```diff
@@ -158,7 +158,7 @@ ${NXT_NODE_VERSION_FILE}: ${NXT_VERSION_H}
 	$echo '#define NXT_NODE_VERNUM \$(NXT_VERNUM)' > $NXT_NODE_VERSION_FILE
 
 ${NXT_NODE_TARBALL}: ${NXT_NODE}-copy
-	sed -e 's/"version"\s*:.*/"version": "\$(NXT_VERSION)",/' \
+	sed -e 's/"version"[[:space:]]*:.*/"version": "\$(NXT_VERSION)",/' \
```

Comment:
I suggest to use something like:
```bash
sed -e "s|%%VERSION%%|\$(NXT_VERSION)|"
```
and replace `1.0.0` in the `package.json` with a distinct placeholder like `%%VERSION%%`.

Reply:
That looks good to me.